### PR TITLE
refactor the user hud add some log

### PR DESCRIPTION
**Pull request by LeviUniAcc**
-also refacotored a little bug, that the curser is not visible after the game ends

### CODE REVIEW

File: Assets/AdditionalScripts/Start Screen Scripts/StartButtonScript.cs [hunk 1]
```diff
@@ -5,9 +5,12 @@
 
 public class StartButtonScript : MonoBehaviour
 {
-    public void LoadScene(string sceneName)
+    public void LoadScene()
```
Comment (marlau97) [inline Assets/AdditionalScripts/Start Screen Scripts/StartButtonScript.cs:8]:
```suggestion
    public void LoadGame()
```
method should be called LoadGame() like it is in the PlayAgainButtonScript

File: Assets/AdditionalScripts/EndScreen/PlayAgainButtonScript.cs [hunk 2]
```diff
@@ -8,8 +8,11 @@ public class PlayAgainButtonScript : MonoBehaviour
     public void LoadGame(string sceneName)
```
Comment (marlau97) [inline Assets/AdditionalScripts/EndScreen/PlayAgainButtonScript.cs:8]:
```suggestion
    public void LoadGame()
```
remove useless parameter

**Review by marlau97 — CHANGES_REQUESTED** [review]
Keep the methods consistent

File: Assets/AdditionalScripts/EndScreen/PlayAgainButtonScript.cs (hunk 2 above)
Comment (LeviUniAcc) [reply Assets/AdditionalScripts/EndScreen/PlayAgainButtonScript.cs:8]: true, ty

File: Assets/AdditionalScripts/Start Screen Scripts/StartButtonScript.cs (hunk 1 above)
Comment (LeviUniAcc) [reply Assets/AdditionalScripts/Start Screen Scripts/StartButtonScript.cs]: true, ty

**Review by marlau97 — APPROVED** [review]

**Review by Gr33ndev — APPROVED** [review]

**Comment by delvh** [comment]
Closes https://github.com/Gamify-IT/issues/issues/44.

**Pull request merged**